doc/classes/Image.xml

```diff
@@ -612,6 +614,7 @@
 			OpenGL texture format [code]RGBA[/code] with four components, each with a bitdepth of 4.
 		</constant>
 		<constant name="FORMAT_RGB565" value="7" enum="Format">
+			OpenGL texture format [code]RGB[/code] with three components. Red and blue have a bitdepth of 4, and green has a bitdepth of 5.
```

Comment:
We don't necessarily need to address this in this PR. But these descriptions should not say "OpenGL texture format"

Reply:
I'm going to keep it in mind myself, but quoting myself from another PR:

> I could have rewritten them outright, but the time I spend on these is directly proportional to how long it will take for them to be reviewed and merged.

Reply:
That's fine. Its more of a note for the future